.devcontainer/docker-compose.yml

```diff
@@ -31,6 +31,17 @@ services:
       - postgres
     command: /bin/sh -c "/start --single-seed-e2e"
 
+  phppgadmin:
+    image: docker.io/bitnami/phppgadmin:7
+    ports:
+      - '8090:8080'
+    environment:
+      DATABASE_HOST: postgres
+      POSTGRESQL_USERNAME: postgres
+      POSTGRESQL_PASSWORD: postgres
+    depends_on:
+      - postgres
+
```

Comment:
Do you need this? What do you need that you cant see in the admin pannel?

Reply:
Just in case we want to tinker at DB level. I was looking at the graphql_auth_status table to set users as verified - might as well have this in our dev container in case we need it later

Reply:
Also I wanted to try and find an email token to test with, but apparently there isn't a table for it?

Reply:
Looks like the token is pseudo-random so there's no need to store it
```
def get_token(user, action, **kwargs):
    username = user.get_username()
    if hasattr(username, "pk"):
        username = username.pk
    payload = {user.USERNAME_FIELD: username, "action": action}
    if kwargs:
        payload.update(**kwargs)
    token = signing.dumps(payload)
    return token
```

Reply:
I've added the `UserStatus` model to the admin panel so verifying should be possible from there now.